File: backend/components/metiers/administration_publique/fixtures/mocks/samples/sample_fixture.test.js
// sample_fixture.test.js - Test unitaire pour sample_fixture.js (mocks/samples)
const sampleFixture = require('./sample_fixture');
describe('mocks/samples/sample_fixture.js', () => {
  it('doit exposer un objet sample correct', () => {
    expect(sampleFixture.sample).toBeDefined();
    expect(sampleFixture.sample.name).toBe('Sample Mock');
    expect(sampleFixture.sample.type).toBe('mock');
    expect(sampleFixture.sample.status).toBe('active');
  });
});

File: backend/components/metiers/administration_publique/fixtures/mocks/samples/sample_fixture.test.py
"""
Test unitaire pour sample_fixture.py (mocks/samples)
"""
from .sample_fixture import sample_mock_fixture

def test_sample_mock_fixture():
    fixture = sample_mock_fixture()
    assert isinstance(fixture, dict)
    assert fixture["name"] == "Sample Mock"
    assert fixture["type"] == "mock"
    assert fixture["status"] == "active"
